eva_submission/nextflow/accession_and_load.nf

```diff
@@ -123,15 +125,17 @@ workflow {
                 .map{row -> tuple(file(row.vcf_file).name, file(row.vcf_file), row.assembly_accession, row.aggregation, file(row.fasta), file(row.report))}
                 .combine(normalise_vcf.out.vcf_tuples, by:0)     // Join based on the vcf_filename
                 .map {tuple(it[0], it[6], it[2], it[3], it[4], it[5])}   // vcf_filename, normalised vcf, assembly_accession, aggregation, fasta, report
-            accession_vcf(normalised_vcfs_ch)
+            chunk_vcf(normalised_vcfs_ch)
+            accession_vcf(chunk_vcf.out.chunked_vcfs.transpose())
             qc_accession_vcf(accession_vcf.out.accession_done)
             sort_and_compress_vcf(qc_accession_vcf.out.qc_accession_done)
-            csi_vcfs = sort_and_compress_vcf.out.compressed_vcf
-            accessioned_files_to_rm = accession_vcf.out.accessioned_filenames
-            all_accession_complete = sort_and_compress_vcf.out.compressed_vcf.collect()
+            all_compressed = sort_and_compress_vcf.out.compressed_vcf.groupTuple(by: 0)
```

Comment:
Following on what Tim mentioned this morning, I'm not sure this will preserve the ordering of the chunks after they've been processed in parallel, we might need to sort them explicitly. Since the filenames contain a chunk ID, it should be doable.